components/SideNavigation.tsx

```diff
@@ -6,37 +6,31 @@ import { useSiteNavigation } from '@/hooks/server';
 import type { NavigationKeys } from '@/types';
 
 type SideNavigationProps = {
-  navigationKey: NavigationKeys;
+  navigationKeys: NavigationKeys[];
   context?: Record<string, RichTranslationValues>;
 };
 
 const SideNavigation: FC<SideNavigationProps> = ({
-  navigationKey,
+  navigationKeys,
   context,
 }) => {
   const { getSideNavigation } = useSiteNavigation();
 
-  const [[, sideNavigationItems]] = getSideNavigation([navigationKey], context);
+  const sideNavigation = getSideNavigation(navigationKeys, context);
+
+  const mapItems = (items: ReturnType<typeof getSideNavigation>) => {
+    return items.map(([, { link, label, items }]) => (
+      <li key={link}>
+        {link ? <ActiveLink href={link}>{label}</ActiveLink> : label}
+
+        {items && items.length > 0 && <ul>{mapItems(items)}</ul>}
```

Comment:
```suggestion
        {!!items?.length && <ul>{mapItems(items)}</ul>}
```

Reply:
I feel that the current one might be easier to read/understand for newcomers, no?

Reply:
@JakobJingleheimer can I resolve this thread?